src/PropertySuggester/Suggesters/SimpleSuggester.php

```diff
@@ -169,7 +170,15 @@ public function suggestByItem( Item $item, $limit, $minProbability, $context ) {
 					);
 				}
 
-				$numericEntityId = $dataValue->getEntityId()->getNumericId();
+				$entityId = $dataValue->getEntityId();
+
+				if ( !( $entityId instanceof Int32EntityId ) ) {
```

Comment:
It does make a little bit more sense, I feel, to check for `ItemId` here, because the `buildTupleCondition` call below expects the entity type to be Item.